### PR TITLE
Bump up the console text size to 20

### PR DESCRIPTION
**Pull request by ashifolfi**
The current font size is a bit hard to read for me. So I bumped it up to make it easier to read.
Feel free to knock it down a few if it's a bit too large.

comparison
image
image

### CODE REVIEW

**Comment by AWildErin** [comment]
Okay, I see https://github.com/ChaosInitiative/Portal-2-Community-Edition/issues/913 affects new installs too

**Comment by hexaflexahexagon** [comment]
to me it looks like the font is contributing more to that screenshot being hard to read, maybe we should wait on this until after we get that fixed & back to something monospaced to see if it's still necessary?

**Comment by AWildErin** [comment]
> to me it looks like the font is contributing more to that screenshot being hard to read, maybe we should wait on this until after we get that fixed & back to something monospaced to see if it's still necessary?

Yeah, I'm not quite sure what's going on with the font, and nobody else who has had the problem (mostly team members) seems to have no idea what's going on either. Bit of a tricky one, since it works for like 95% of people

**Comment by ashifolfi** [comment]
it might be worthwhile to add some sort of fix or maybe the ability to easily increase console font size without editing the panorama files until you can figure out the font issue

**Comment by craftablescience** [comment]
im fine with bumping up the font size a bit, even with the correct font its really small for me

File: styles/console.sass [hunk 1]
```diff
@@ -33,7 +33,7 @@
 	> MultiselectLabel
 		text-shadow-fast: none
 		font-family: monospaceFont
-		font-size: 12px
+		font-size: 20px
```
Comment (craftablescience) [inline styles/console.sass:36]:
```suggestion
		font-size: 16px
```

File: styles/console.sass [hunk 2]
```diff
@@ -54,7 +54,7 @@
 	color: rgb(255, 123, 2)
 
 ConsoleInput
-	font-size: 16px
+	font-size: 20px
```
Comment (craftablescience) [inline styles/console.sass:57]:
```suggestion
	font-size: 16px
```

**Review by craftablescience — CHANGES_REQUESTED** [review]
i checked ingame and 20px is MASSIVE, mind if its bumped down to 16?

ill add "console text size" to the pano settings issue on gh as well

**Review by craftablescience — APPROVED** [review]

**Pull request merged**